contents/blog/best-open-source-analytics-tools.md

```diff
@@ -11,221 +11,301 @@ featuredImageType: full
+Plausible Analytics is available under a AGPLv3 license and can be self-hosted via Docker. Pricing for its hosted version starts at €9 per month for 10,000 monthly pageviews.
 
-### Open source license
-Plausible Analytics is available under a AGPLv3 license. Pricing depends on the amount of traffic, with a 30-day free trial available. Check [the Plausible Analytics repo on GitHub](https://github.com/plausible/analytics/) for more info.
+> **Also consider:** Plausible is one of a large subset of privacy-first analytics products that have proven popular in the wake of GDPR and less permissive attitudes to user tracking. [Umami](https://umami.is/) is another popular, open-source product in this space. Read our [GDPR analytics guide](/blog/best-gdpr-compliant-analytics-tools) for more options.   
 
-## 6. GrowthBook
+### GrowthBook
 
 ![GrowthBook - best open source analytics tools](../images/blog/open-source-testing-tools/growthbook.png)
 
+- **GitHub Stars:** 3.5k
+- **Languages:** TypeScript (92%)
+- **Alternative to:** LaunchDarkly
+- [GrowthBook on GitHub](https://github.com/growthbook/growthbook)
+
 GrowthBook is an [open source A/B testing](/blog/best-open-source-ab-testing-tools) and feature flag tool for teams who want granular control over rolling out new features, and the ability to validate those changes through experiments. 
 
 It supports multiple development environments, each with a unique API endpoint, and its comprehensive feature flag functionality allows for targeting based on user properties.
 
-You don't need to import metric data into GrowthBook as it works with your data wherever it lives, be that a data warehouse or a analytics tool like Mixpanel, but it lacks any additional analytics features.
+You don't need to import metric data into GrowthBook as it works with your data wherever it lives, be that a data warehouse or an analytics tool like Mixpanel, but it lacks any additional analytical features.
 
-### Who is GrowthBook for?
+#### Who is GrowthBook for?
 
-GrowthBook is ideal for engineering teams who want flexible, instant control over what users see in their product and already have, or don't require, a broader analytics platform. It can be self-hosted free-for-life, but there's a cloud option as well which is free for up to five seats, then $20 per seat per month thereafter.
+GrowthBook is ideal for engineering teams who want flexible, instant control over what users see in their product and already have, or don't require, a broader analytics platform.
 
-### Strengths
+#### Strengths
 - Multivariate feature flags
 - Support for Mixpanel JQL querying
 - A/B testing visual editor
 - Can be self-hosted
 - No performance impact
 
-### Open source license
+#### Open source license and monetization
+
+GrowthBook is distributed under an MIT license and self-hosted is supported at no cost. The cloud version is free for up to three users, and $20 per user per month thereafter. 
+
+## The best open source data and BI tools
```

Comment:
How come it's not better to split this into a second article?

Reply:
I might at some point, but this article is already ranking for similar terms so I'm trying to boost it on those. Main target is to ranke for 'open source data tools' as well, hence the headline change.